### DISCUSSION TITLE
I've Compiled for 5.3.0 production

### DISCUSSION THREAD

**Discussion by courcreate** [microdee/OpenUnrealSpaceMouse#99] [Show and tell]
[sPACEMOUSE.zip](https://github.com/microdee/UE4-SpaceMouse/files/12832340/sPACEMOUSE.zip)

**Comment by microdee** [comment]
congrats, did you change anything? if so can you make a PR out of it?

**Reply by courcreate** [reply]
I just had to install the correct visual studio versions. 5.3.0 requires the latest in studio. So all of the SDKs for game making have to be installed or it won't compile correctly. Other than that I have no idea how to make a pull request or anything else because I don't use GitHub often. I just had to learn how to do this because the versions weren't working for me and I have worked that I need to get done so I researched how to run the UAT bat to update this plugin. I haven't changed anything so feel free to make your own request and even take credit for this.

**Reply by onyxlee** [reply]
Thanks, this release works great with 5.3.1.

**Comment by OURLEDS** [comment]
The plugin is missing in Project Settings - Plugins, canit change any settings.
Testing on 5.3.2

The plugin is working, but I can't adjsut any settings.


On the right folder:
![imagen](https://github.com/microdee/UE4-SpaceMouse/assets/46494097/589b40e5-901e-401e-98fb-6ab226c65bb6)

Activated:
![imagen](https://github.com/microdee/UE4-SpaceMouse/assets/46494097/df68c8a5-2d02-41ac-be68-9f0c5ed7e668)

Missing from list of plugins:
![imagen](https://github.com/microdee/UE4-SpaceMouse/assets/46494097/49dcbdb9-61af-45c6-8d61-adc1079a8a12)
